package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "kyber-utils-sc",
+  "version": "2.0.0",
+  "description": "This repository has the utils contracts for the Kyber protocol.",
+  "main": "truffle.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "./test.sh",
+    "t": "./test.sh",
+    "compile": "./compile.sh",
+    "c": "./compile.sh",
+    "compileSol6": "./compileSol6.sh",
+    "coverage": "./coverage.sh",
+    "lint": "yarn solhint 'contracts/**/*.sol' && yarn prettier -c './**/*.js'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/KyberNetwork/kyber_utils_sc.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/KyberNetwork/kyber_utils_sc/issues"
+  },
+  "homepage": "https://github.com/KyberNetwork/kyber_utils_sc#readme",
+  "dependencies": {
+    "chai-as-promised": "7.1.1",
+    "compare-versions": "3.6.0",
```

Comment:
please make sure we use all packages.

I think we don't use ethers.
also not sure about compare-versions.
also truffle-assertions.
truffle-flattener
winston (its for fuzzer tests)
compare-versions.